WebRoot/js/zimbraMail/prefs/view/ZmSharingPage.js

```diff
@@ -921,3 +928,17 @@ function(share, list, compareFunc) {
 	}
 	return null;
 };
+
+//overriding no results container getter function for custom behavior
+ZmSharingListView.prototype._getNoResultsContainer =
+function() {
+	var div = document.createElement("div");
+	div.className = "ZmListViewTableNoResultsContainer";
+	return div;
+};
+
+//preventing selection in the list view.
+ZmSharingListView.prototype._allowLeftSelection =
```

Comment:
This is an option that is exposed in DwtListView, so we have overridden that.
It says:
	// overload me (and return false) if you dont want to actually select clickedEl